README.md

```diff
@@ -24,6 +24,7 @@ The following utilities are prerequisites to installing F´:
 - git
 - Python 3.5+ with pip
 
+
```

Comment:
This line should go away (I personally prefer squashing and force pushing, you only need one commit for this)

Reply:
You should still remove this line.

And, I'd really encourage you to squash all your commits into a single commit w/ a single description of what you're actually doing to the INSTALL file.